src/App.tsx

```diff
@@ -26,70 +27,132 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  visibleGoods.sort((a, b) => {
```

Comment:
I think it's better to avoid using names like 'a' or 'b', because you actually sort goods here

Reply:
Sort is always used with (a, b) parameters, it's a kind of commonplace. I would be more confused with the names like 'good1' and 'good2'. It would be less readable in my opinion, but I'm ready to change the names if menthor says so.